docs/requirements.txt

```diff
@@ -1,7 +1,7 @@
 backports-abc==0.5
 Click==7.0
 futures==3.2.0
-Jinja2==2.10
+Jinja2>=2.10.1
```

Comment:
want to use `==` so the docs build the same everywhere?

Reply:
sure